metrics-delivery-endpoint-api/src/main/java/eu/openaire/mas/delivery/MetricEntry.java

```diff
@@ -7,28 +7,28 @@
  */
 public class MetricEntry {
 
-    private final String groupId;
+    private final String resourceId;
 
-    private final String metricId;
+    private final String kpiId;
     
-    private final Object value;
+    private final float value;
```

Comment:
OK, so we are going to provide just a single value regardless the KPI type (representing either state or events), right? No time series etc. We'd just need to distinguish the KPI type on our end in order to know whether to provide the cumulative metric over a period of time (events) or just a most recently registered value (state).

So I assume we are not going to represent the KPIs using graphic representation on a receiving end, just plain numbers. Otherwise the time series resolution would be bound to the intervals of API querying (I will ask explicitly Antonis in the ticket).